### PR TITLE
chore(github): Updated security documentation and CLOMonitor exemptions

### PR DESCRIPTION
**Pull request by eddie-knight**
Checklist:

* [ ] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [ ] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [ ] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [ ] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Review by jmeridth — APPROVED** [review]
Thank you

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ Any breaking changes to a chart (backwards incompatible) require:
 
 ### New Application Versions
 
+Helm charts are intended to be created for all relevant releases (Argo CD, Workflows, Rollouts, Events). Associated dependencies, such as Redis, will use the version recommended by the associated release.
```
Comment (mkilchhofer) [inline CONTRIBUTING.md:50]: What does that mean "for all relevant releases"? Right now we cannot release a new chart version for a patch happening on the non-mainline version.

Comment (eddie-knight) [reply CONTRIBUTING.md:50]: @mkilchhofer or @jmeridth, could you suggest a revision to increase the precision here? To be honest I was being somewhat vague in summarizing thoughts that JM shared offline

Comment (jmeridth) [reply CONTRIBUTING.md:50]: s/for all relevant releases/for all non-patched releases

**Review by jmeridth — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (eddie-knight) [reply CONTRIBUTING.md]: applied, and went back now to push not-from-mobile for DCO

**Pull request merged**